docs/format.rst

```diff
@@ -292,3 +292,36 @@ The commit process is as follows:
 When checking whether two transactions conflict, be conservative. If the
 transaction file is missing, assume it conflicts. If the transaction file 
 has an unknown operation, assume it conflicts.
+
+
+External Manifest Store
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If the backing object store does not support \*-if-not-exists operations, an
+external manifest store can be used to allow concurrent writers. An external
+manifest store is a KV store that supports put-if-not-exists operation.
```

Comment:
Just want to be clear that this isn't intended as a replacement for the manifests in object storage.
```suggestion
manifest store is a KV store that supports put-if-not-exists operation. The
external manifest store supplements but does not replace the manifests in
object storage. A reader unaware of the external manifest store could read a
table that uses it, but it might be up to one version behind the true latest
version of the table.
```